vllm/v1/metrics/loggers.py

```diff
@@ -185,6 +185,22 @@ def __init__(self,
         #
         # Scheduler state
         #
+        gauge_forward_cpu_time_ms = self._gauge_cls(
```

Comment:
these should be counters

Reply:
Actually histograms I think?

Reply:
actually I never understand the difference between gauge and counter lol. what's the difference?

Reply:
Gauge tracks a point-in-time level, like used memory. Counter tracks the number of occurrences of some event over time, like requests processed. Histogram is for tracking occurrences of some event that each have an associated magnitude, like response times or most other timings.

Reply:
hm yeah I actually wanted point.

Reply:
But you are measuring/recording the wall time taken for each forward pass? So it should be histogram I think?

Also I think it should be called clock time or wall time rather than cpu time. It might be good to separately record cpu time (both for the model forward thread and the process as a whole).